src/test/java/de/uhd/ifi/se/decision/management/jira/view/TestRequirementsDashboardItem.java

```diff
@@ -0,0 +1,60 @@
+package de.uhd.ifi.se.decision.management.jira.view;
+
+import static org.junit.Assert.assertNotNull;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.atlassian.jira.component.ComponentAccessor;
+import com.atlassian.jira.issue.issuetype.IssueType;
+import com.atlassian.jira.issue.issuetype.MockIssueType;
+import com.atlassian.jira.mock.MockProjectManager;
+import com.atlassian.jira.project.MockProject;
+import com.atlassian.jira.project.Project;
+
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.config.PluginInitializer;
+import net.java.ao.test.jdbc.NonTransactional;
+
+
+
+public class TestRequirementsDashboardItem {
+	
+	private RequirementsDashboardItem dashboardItem;
+	private Map<String, Object> params;
+	private MockProjectManager projectManager;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'projectManager' could be replaced by a local variable.](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4584657)